pkg/controllers/user/controllers.go

```diff
@@ -92,6 +92,7 @@ func RegisterFollower(ctx context.Context, cluster *config.UserContext, kubeConf
 	cluster.RBAC.ClusterRoleBindings("").Controller()
 	cluster.RBAC.RoleBindings("").Controller()
 	cluster.Core.Endpoints("").Controller()
+	workload.NewWorkloadController(ctx, cluster.UserOnlyContext(), nil)
```

Comment:
Please can you explain why this change is needed?

Reply:
Query graph API in project level need to get workload info, for rancher deployed in HA mode, follower nodes don't have workload info in the cache, so we add it workload here.